openmc/model/model.py

```diff
@@ -398,8 +443,30 @@ def deplete(self, timesteps, method='cecm', final_step=True,
                 depletion_operator.cleanup_when_done = True
                 depletion_operator.finalize()
 
-    def export_to_xml(self, directory='.', remove_surfs=False):
-        """Export model to XML files.
+    def export_to_xml(self, directory='.', remove_surfs=False, separate_xmls=True):
```

Comment:
Any thoughts on if/when we should make a single XML file default?

Reply:
I didn't to it here because it would require a huge change to the test suite. There may be ways around that though. I'll give it some more thought.